spec/server/ServerRendererSpec.js

```diff
@@ -181,7 +181,7 @@ describe("ServerRenderer", function() {
     describe("when layout does NOT have a body", function() {
 
         beforeEach(function() {
-            layout.el.innerHTML = "<html><head></head></html>";
+            layout.$("body").remove();
```

Comment:
this case should probably test if a user specifies a template without a body rather than setting up the case